7zip/tests/test.bats

```diff
@@ -5,8 +5,8 @@ expected_version="$(echo $TEST_PKG_IDENT | cut -d/ -f 3)"
 }
 
 @test "7z exe output mentions expected version $expected_version" {
-  run hab pkg exec $TEST_PKG_IDENT 7z
-  [[ "$output" =~ 7-Zip\ \[64\]\ $expected_version ]]
+   run hab pkg exec "${TEST_PKG_IDENT}" 7z
+   echo ${output} | grep -Eo "7-Zip \[64\] $expected_version"
```

Comment:
You might consider using [here doc](http://tldp.org/LDP/abs/html/here-docs.html) syntax instead of ` echo | `.  It looks a little funky at first but I find it a little more legible.

`grep -Eo "7-Zip \[64\] $expected_version" <<< "$output"`

Reply:
Thanks @smacfarlane!  Refactored and tests all working.